modules/administration-guide/partials/snip_preparing-images-for-a-restricted-environment.adoc

```diff
@@ -0,0 +1,82 @@
+// Included in:
+// modules/administration-guide/pages/installing-che-in-a-restricted-environment.adoc
+// modules/administration-guide/pages/upgrading-che-using-the-cli-management-tool-in-restricted-environment.adoc
+
+
+:my_internal_registry: __<my_internal_registry>__
+
+* The destination OpenShift cluster is ready to operate on a restricted network. See link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-restricted-networks.html[OpenShift documentation - Using Operator Lifecycle Manager on restricted networks].
+
+* An administrative access to the destination OpenShift cluster and the pass:c,a,q[{my_internal_registry}] destination registry.
+
+* The `{prod-cli}` management tool for {prod-short} version {prod-ver} is available on your workstation. See xref:using-the-chectl-management-tool.adoc[].
+
+* The `curl`, `jq`, and `yq` tools are available on your workstation.
```

Comment:
There are 2 `yq` that have different syntax: https://github.com/mikefarah/yq and https://github.com/kislyuk/yq. Which one should I install?

Reply:
yq requirement is gone!